examples/jumbo.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"github.com/intel-go/nff-go/flow"
+	"github.com/intel-go/nff-go/packet"
+	"time"
+)
+
+// You need to define MEMORY_JUMBO in low/low.h for maxPacketSegment more than 2014.
+// Explanation: internal packet buffers are 2048 bytes, first packet will have
```

Comment:
Shouldn't this comment be phrased differently with initialization flags instead of defines?